File: SDKs/Aspose.Pdf_Cloud_SDK_for_.NET/src/Com/Aspose/PDF/Model/TextReplaceListRequest.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class TextReplaceListRequest {
    public List<TextReplace> TextReplaces { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class TextReplaceListRequest {\n");
      sb.Append("  TextReplaces: ").Append(TextReplaces).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Pdf-Cloud-SDK-for-Java/src/main/java/com/aspose/pdf/model/TextReplaceListRequest.java
package com.aspose.pdf.model;

import java.util.*;
public class TextReplaceListRequest {
  private List<TextReplace> TextReplaces = new ArrayList<TextReplace>();
  /**
	 * getTextReplaces
	 * Gets List<TextReplace>
	 * @return TextReplaces
	 */
  public List<TextReplace> getTextReplaces() {
    return TextReplaces;
  }

	/**
	 * setTextReplaces
	 * Sets List<TextReplace>
	 * @param TextReplaces List<TextReplace>
	 */
  public void setTextReplaces(List<TextReplace> TextReplaces) {
    this.TextReplaces = TextReplaces;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class TextReplaceListRequest {\n");
    sb.append("  TextReplaces: ").append(TextReplaces).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

